editor.planx.uk/src/@planx/components/PropertyInformation/Public.tsx

```diff
@@ -248,12 +217,12 @@ function PropertyDetails(props: PropertyDetailsProps) {
   };
 
   return (
-    <PropertyDetailsList component="dl">
+    <SummaryListTable showChangeButton={true}>
       {filteredData.map(({ heading, detail, fn }: PropertyDetail) => (
         <React.Fragment key={heading}>
-          <Box component="dt">{heading}</Box>
-          <Box component="dd">{detail}</Box>
-          <Box component="dd">
+          <dt>{heading}</dt>
+          <dd>{detail}</dd>
```

Comment:
nit: `<Box component="dt">` just compiles down to a `dt` so should be the same from an accessibility point of view.

Using a `Box` and setting a `component` prop is more consistent with the rest of the code base and would more easily allow us to customise this if needed (e.g. using the `sx` prop).

Reply:
Thanks for the context on this, I've reverted and updated other instances of description lists to be consistent.